### ISSUE TITLE
Volume and Mute/Unmute not working

### ISSUE THREAD

**Issue by sander-p** [vdesabou/alfred-spotify-mini-player#47]
Using Alfred Remote, the Volume Up, Volume Down and Mute/Unmute triggers are not working at all for me. Everything else seems to be working fine.

Any clues?

Cheers,

Sander

**Comment by vdesabou** [comment]
Can you run in a terminal, the following command:

```
osascript -e 'set volume output volume (output volume of (get volume settings) + 6)'
```

And copy/paste output here?

This should normally increase the volume

**Comment by sander-p** [comment]
That gives: Can’t make missing value into type number. (-1700)

If I switch from Digital Out to Line Out, that does change the volume. Makes sense: if I use Digital Out, I can't adjust the output volume at all -- not with the slider in the Sound preference pane, or with the media keys.

But I don't want to change the mac settings, I want to change the Spotify volume slider. Maybe it's better so simply pass along keyboard shortcuts to the Spotify app? Cmd-up arrow and cmd-down arrow work fine.

**Comment by vdesabou** [comment]
Ok thanks for the investigation.

>But I don't want to change the mac settings, I want to change the Spotify volume slider. Maybe it's better so simply pass along keyboard shortcuts to the Spotify app? Cmd-up arrow and cmd-down arrow work fine.

I decided to control system volume instead of Spotify volume (commit 3d43bdd55d6637a137a5da9aaaa5c939d8971296) because I used the the workflow Remote triggers to control system volume. But it is true that it makes sense to control Spotify volume only.. I think I will change it in a near future.

So stay tuned for a future update!
